client/src/pages/ChatsBlock.jsx

```diff
@@ -0,0 +1,297 @@
+import { useContext } from 'react';
+import { userService } from '../services/userService.js';
+import { Formik, Form, Field } from 'formik';
+import { Context } from '../context/context.jsx';
+import { useNavigate } from 'react-router-dom';
+
+function chatNameValidation(newChatName) {
+  if (!newChatName || newChatName.trim().length === 0) {
+    return 'Chat name required';
+  }
+
+  if (newChatName.trim().length < 2) {
+    return 'Chat name is too short';
+  }
+
+  if (newChatName.length > 30) {
+    return 'Less then 30 characters required';
```

Comment:
There is a typo in the error message: 'Less then 30 characters required' should be 'Less than 30 characters required'.